File: summary/foreach_multiple.swift
var arr:[[String]] = [["Hello","Go"],["Bye","Python"]]
print(arr)
print("================")
arr.forEach { v in
    print(v)
    print(arr.count)
}
print("================")
arr.forEach { v in
    print(v[0]+v[1])
}

// swift foreach.swift
//
// [["Hello", "Go"], ["Bye", "Python"]]
// ================
// ["Hello", "Go"]
// 2
// ["Bye", "Python"]
// 2
// ================
// HelloGo
// ByePython

File: summary/foreach_multiple.go
package main
import "fmt"

func main() {
  arr := [][]string{
    {"Hello", "Go"},
    {"Bye", "Python"},
  }
  fmt.Println(arr)
  fmt.Println("================")
  for i, v := range arr {
    fmt.Println(i, v)
    if(len(arr)-1 == i){
      fmt.Println("================")
    }
  }
  for i, v := range arr {
    fmt.Println(i, v[0], v[1])
  }
}

// go run func_multiple.go
//
// [[Hello Go] [Bye Python]]
// ================
// 0 [Hello Go]
// 1 [Bye Python]
// ================
// 0 Hello Go
// 1 Bye Python